kolona/task.py

```diff
@@ -118,14 +138,23 @@ def can_retry(self):
         return self.retry_attempt < self.max_retries
 
 
-def task(*args, queue=None, max_retries=3, retry_intervals=None, **kwargs):
+def task(
```

Comment:
I just removed *args and **kwargs here, because they were not used. Did you have any plans with them?

Reply:
No, given all tests pass w/o them it's totally ok to remove them.